### PR TITLE
docs: mention tooling to retrieve installation token for GitHub platform

### PR DESCRIPTION
**Pull request by MShekow**
## Changes

Updates the GitHub platform docs, mentioning tooling to retrieve the GitHub installation token (as it expires after one hour).

## Context

Addresses https://github.com/renovatebot/renovatebot.github.io/issues/288

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [ ] No documentation update is required
- [x] This is a docs _only_ update 

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/platform/github/index.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ Any tokens that do not start with `ghs_` (for example tokens from GitHub Enterpr
 !!! note
     The installation tokens expire after 1 hour and need to be regenerated regularly.
     Alternatively as environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
+    There are tools to regenerate the installation token. If you run your self-hosted Renovate Bot in a GitHub Actions workflow, you can use the actions [`jnwng/github-app-installation-token-action`](https://github.com/jnwng/github-app-installation-token-action) or [`tibdex/github-app-token`](https://github.com/tibdex/github-app-token). Other tools are [github-app-installation-token](https://github.com/gagoar/github-app-installation-token) (Node.js libary / CLI) or the [`mshekow/github-app-installation-token`](https://github.com/MShekow/github-app-installation-token) Docker image.
```
Comment (HonkingGoose) [inline lib/modules/platform/github/index.md:74]:
```suggestion

<!-- prettier-ignore -->
!!! tip "Third-party tools to regenerate installation tokens"
    If you're self-hosting Renovate within a GitHub Actions workflow, then you can use the [`jnwng/github-app-installation-token-action`](https://github.com/jnwng/github-app-installation-token-action) or [`tibdex/github-app-token`](https://github.com/tibdex/github-app-token) actions.
    If you use Node.js/CLI, then you can use the [`github-app-installation-token`](https://github.com/gagoar/github-app-installation-token) package/library.
    If you use Docker, then you can use the [`mshekow/github-app-installation-token`](https://github.com/MShekow/github-app-installation-token) image.
```

Changes:

- Make a new `tip` admonition for the whole block
- Put first sentence in the `tip` title
- Convert rest of text to match the "If you use ..., then you can use ..." pattern
- Use `monospace` for all "package names"
- Fixed typo in word `libary` -> `library`

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: lib/modules/platform/github/index.md [hunk 2]
```diff
@@ -72,6 +72,12 @@ Any tokens that do not start with `ghs_` (for example tokens from GitHub Enterpr
     The installation tokens expire after 1 hour and need to be regenerated regularly.
     Alternatively as environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
 
+<!-- prettier-ignore -->
+!!! tip "Third-party tools to regenerate installation tokens"
+    If you're self-hosting Renovate within a GitHub Actions workflow, then you can use the [`jnwng/github-app-installation-token-action`](https://github.com/jnwng/github-app-installation-token-action) or [`tibdex/github-app-token`](https://github.com/tibdex/github-app-token) actions.
+    If you use Node.js/CLI, then you can use the [`github-app-installation-token`](https://github.com/gagoar/github-app-installation-token) package/library.
```
Comment (HonkingGoose) [inline lib/modules/platform/github/index.md:78]: What's the best word here? Is it package or library, or both?

File: lib/modules/platform/github/index.md [hunk 3]
```diff
@@ -72,6 +72,12 @@ Any tokens that do not start with `ghs_` (for example tokens from GitHub Enterpr
     The installation tokens expire after 1 hour and need to be regenerated regularly.
     Alternatively as environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
 
+<!-- prettier-ignore -->
```
Comment (HonkingGoose) [inline lib/modules/platform/github/index.md:75]: The maintainers should probably review the third-party actions? I think we should be careful with our recommendations in the docs.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: lib/modules/platform/github/index.md (hunk 2 above)
Comment (rarkins) [reply lib/modules/platform/github/index.md:78]: Let's go with package

Comment (rarkins) [inline lib/modules/platform/github/index.md:78]:
```suggestion
    If you use Node.js/CLI, then you can use the [`github-app-installation-token`](https://github.com/gagoar/github-app-installation-token) package.
```

**Review by rarkins — CHANGES_REQUESTED** [review]

**Review by rarkins — APPROVED** [review]

**Review by HonkingGoose — APPROVED** [review]

**Comment by MShekow** [comment]
Thank you both for adapting the changes :rocket:. Is there anything left to do before merging?

**Pull request merged**